.github/workflows/npm-release.yml

```diff
@@ -55,30 +55,13 @@ jobs:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
   containers:
     if: github.repository == 'CycloneDX/cdxgen'
-    runs-on: ubuntu-latest
+    runs-on: ["self-hosted", "ubuntu", "amd64"]
```

Comment:
Going to merge this PR to test the performance of building the all-in-one image.